### ISSUE TITLE
More requests - faster moving left to right etc

### ISSUE THREAD

**Issue by davodogster** [Dubrzr/SignalAnnotation#4]
Hi Julien @Dubrzr. I've been successfully uploading my signals to the app.html. A couple of issues which mean that I will have to use a slower, different annotation tool for the time being: The main one being that I'm using a large signal so the left right movement is far too slow, hence my suggestion on a previous issue about clicking on the signal and just dragging it left or right as far as you want, would be extremely helpful. Also, it failed to upload a signal of length 200,000 (but 100,000 worked). It's a problem for me because my signals are 200k - 500k in length. I could trim them or resample them if desperate but would prefer not to.

Best regards, Sam

Edit:
Have recently discovered you can increase these values in app.html to make the left to right movements bigger :)
![image](https://user-images.githubusercontent.com/46079516/90717239-3dd50880-e303-11ea-9f85-8520684a844d.png)

Would still be good to have click and drag so you can move arbitrary lengths left to right.

**Comment by Dubrzr** [comment]
Hi Sam :D 

Just to let you know that ctrl+zoom works fine on firefox for now, it doesn't zoom in the page too. Wondering why it doesn't on Chrome...

What is the problem with signal of length 200k? The upload, the parsing, displaying it is slow?

Thanks

**Comment by davodogster** [comment]
Hi it would upload but it was just blank.
Tried it with 300k examples and the same thing happened.

Sam
________________________________

**Comment by Dubrzr** [comment]
Yes you can modify the code to move faster, one improvement would be to let users choose the value using a number input form on the ui.

Ok so for the blank problem, it's probably a performance issue, I'll try on my side with bigger examples. 
If that's the case, we need some optimizations...

**Comment by davodogster** [comment]
> Yes you can modify the code to move faster, one improvement would be to let users choose the value using a number input form on the ui.
> 
> Ok so for the blank problem, it's probably a performance issue, I'll try on my side with bigger examples.
> If that's the case, we need some optimizations...

Yes it would be helpful to chose a value in the UI.
The input size might be a chrome issue.. The 200K and 300K long series can be uploaded in Mozilla Firefox - I just tried it. Firefox also fixed the page zooming problem.
